eZ/Publish/Core/Repository/Values/Content/Content.php

```diff
@@ -70,6 +70,14 @@ public function getVersionInfo()
     /**
      * {@inheritdoc}
      */
+    public function getName($languageCode = null)
+    {
+        return $this->versionInfo->getName($languageCode);
```

Comment:
Nit pick: `getVersionInfo()` method is part of abstract. Wouldn't use of method instead of property as an entry point for `versionInfo` be easier to do future refactors?